pkg/collector/corechecks/oracle-dbm/statements.go

```diff
@@ -202,39 +230,40 @@ func ConstructStatementMetricsQueryBlock(sqlHandleColumn string, whereClause str
 func GetStatementsMetricsForKeys[K comparable](c *Check, keyName string, whereClause string, keys map[K]int) ([]StatementMetricsDB, error) {
 	if len(keys) != 0 {
 		db := c.db
-		driver := c.driver
+		//driver := c.driver
 		var bindPlaceholder string
 		keysSlice := maps.Keys(keys)
 
-		if driver == common.Godror {
-			bindPlaceholder = "?"
-		} else if driver == common.GoOra {
-			// workaround for https://github.com/jmoiron/sqlx/issues/854
-			for i := range keysSlice {
-				if i > 0 {
-					bindPlaceholder = bindPlaceholder + ","
+		//if driver == common.Godror {
+		bindPlaceholder = "?"
+		/*
+			} else if driver == common.GoOra {
+				// workaround for https://github.com/jmoiron/sqlx/issues/854
+				for i := range keysSlice {
+					if i > 0 {
+						bindPlaceholder = bindPlaceholder + ","
+					}
+					bindPlaceholder = fmt.Sprintf("%s:%d", bindPlaceholder, i+1)
 				}
-				bindPlaceholder = fmt.Sprintf("%s:%d", bindPlaceholder, i+1)
-			}
-		} else {
-			return nil, fmt.Errorf("statements wrong driver %s", driver)
-		}
+			} else {
+				return nil, fmt.Errorf("statements wrong driver %s", driver)
+			} */
```

Comment:
can we clean this up or is this comment needed?

Reply:
done